src/tooltip.js

```diff
@@ -86,6 +86,10 @@ angular.module('cp-tooltip')
 					tooltipDisplayed = false;
 					if(tooltipEl) tooltipEl.remove();
 				}
+
+				function encodeHTML(str) {
+					return str.replace(/&/g, '&amp;').replace(/</g, '&lt;').replace(/"/g, '&quot;');
+				}
```

Comment:
https://github.com/CanopyTax/workflow-ui/blob/80422938a316523b0a253ecd07d2388473d4e9ac/angular/app/client/general/tasks/tasks.service.js#L7 is a good example of how to do it. You can inject `$sanitize` into this directive.